manifest.json

```diff
@@ -32,7 +32,19 @@
     },
     "projectOptions": ["app", "addon"],
     "nodeVersion": "4.0.0",
-    "commands": ["github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"]
+    "commands": [
+      " github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"
```

Comment:
Can you revert these changes from es5-getter-ember-codemod please?

Reply:
Done